### ISSUE TITLE
Overtime feature

### ISSUE THREAD

**Issue by kevinpapst** [kimai/kimai#144]
Forum post: http://forum.kimai.org/index.php?topic=166.0

[![bountysource](https://api.bountysource.com/badge/issue?issue_id=1909164)](https://www.bountysource.com/issues/1909164)

**Comment by fredericmohr** [comment]
I think this is a great idea and a feature that has been missing in kimai for a really long time. I use kimai to track time on project and customer bases. In the cases where I work a fixed set of hours per day for a customer, I always have to check how many hours I worked during the last month and how many hours I should have worked. It would be super awesome if I could simply specify a number of hours per day/customer and kimai would tell me if I'm currently under or over that amount.

Example:
Customer1: 5h per day
I would want Kimai to calculate my overall time, not just for the selected time frame. Say I've selected the time frame for this week, but I have done major overtime during the last two weeks. I would want kimai to display on the top, that I have already done my hours for the selected (filter) customer. 

In essence, kimai would have to calculate the number of work days since the first entry for that customer (or from a specified "contract start" date), multiply the number of work days with the specified number-of-hours-per-day and substract the recorded number of hours worked for that customer. The difference should then be displayed on top either in +black or -red.

If no customer or project filter is set or the selected customer/project has no hours-per-day value, show a simple "N/A" for overtime. Also, there should be an option for hours-per-day in general, for those who use kimai with only one customer or employee. In this case, it wouldn't matter how many projects you're working on, you have one set of hours/day for all of them combined.

I'm not much of a coder (sorry, I know talk is cheap) but if I can be of any help, I'll gladly do my best. (I also added a bounty to this issue, maybe that'll help a bit)
